stdlib/std/reserve.tact

```diff
@@ -1,2 +1,9 @@
 @name(raw_reserve)
-native nativeReserve(amount: Int, mode: Int);
\ No newline at end of file
+native nativeReserve(amount: Int, mode: Int);
+
+const ReserveExact: Int = 0;
+const ReserveAllExcept: Int = 1;
+const ReserveAtMost: Int = 2;
+const ReserveAddOriginalBalance: Int = 4;
+const ReserveInvertSign: Int = 8;
+const ReserveBounceIfActionFail: Int = 16;
```

Comment:
Currently, neither TVM docs, not Tact docs define the +16 flag (`ReserveBounceIfActionFail`):

<img width="1122" alt="image" src="https://github.com/tact-lang/tact/assets/2316541/d3de31d4-5867-46d5-b60f-6b24806142ec">

Where does it come from?

Reply:
<img width="694" alt="Screenshot 2024-03-25 at 8 08 14 PM" src="https://github.com/tact-lang/tact/assets/42098239/4d89c3aa-afc8-4824-a00e-ad94fee14edf">
https://docs.ton.org/learn/tvm-instructions/tvm-upgrade-2023-07#sending-messages

Reply:
Got it, thanks!